### PR TITLE
Update EIP-7600: Update eip-7600.md

### PR DESCRIPTION
**Pull request by timbeiko**
This PR updates the Pectra Meta EIP to categorize EIPs as defined in [EIP-7723](https://eips.ethereum.org/EIPS/eip-7723). 

Note that given the late addition of these categories to the EIP, and discussions on [ACDE#195](https://github.com/ethereum/pm/issues/1142), I've opted to limit the `Proposed for Inclusion` section to EIPs which have been discussed in the past few ACD calls. If anyone feels something is missing, please propose it as an addition to the PR. 

One notable proposal still missing is the one about [increasing the blob gas reserve price](https://github.com/ethereum/pm/issues/1142#issuecomment-2316630318), which still does not have an EIP drafted.

### CODE REVIEW

**Comment by abcoathup** [comment]
> One notable proposal still missing is the one about [increasing the blob gas reserve price](https://github.com/ethereum/pm/issues/1142#issuecomment-2316630318), which still does not have an EIP drafted.

**EIP7762: Increase Blob Base Fee** has been drafted: #8849

**Comment by timbeiko** [comment]
Thanks @abcoathup, added!

File: EIPS/eip-7600.md [hunk 1]
```diff
@@ -45,16 +45,29 @@ This Meta EIP lists the EIPs formally considered for and included in the Prague/
...
+* [EIP-7742](./eip-7742.md): Uncouple blob count between CL and EL 
+
+### EIPs Proposed for Inclusion
+
+For clarity, and due to the late addition of this section in the Pectra network upgrade planning process, only EIPs which have recently been proposed or discussed are listed here. For the full list of Pectra proposals, please refer to the `discussion-to` link for this EIP. If the author has missed relevant EIPs in this section, please propose additions by opening a pull request.  
+
+* [EIP-6493](./eip-6493.md): SSZ Transaction Signature Scheme
+* [EIP-7495](./eip-7495.md): SSZ StableContainer
+* [EIP-7688](./eip-7688.md): Forward compatible consensus data structures
+* [EIP-7761](./eip-7761.md): IS_CONTRACT instruction
```
Comment (gumb0) [inline EIPS/eip-7600.md:57]: renamed to `HASCODE`

**Review by Magken91 — APPROVED** [review]

**Comment by timbeiko** [comment]
Closed in favor of https://github.com/ethereum/EIPs/pull/8911

**Pull request closed without merging**